Modules/Filtering/FFT/include/itkFFTWCommon.h

```diff
@@ -331,14 +327,12 @@ class Proxy<float>
            ComplexType * out,
            int           sign,
            unsigned int  flags,
-           int           threads = 1,
+           int           itkNotUsed(threads) = 1,
            bool          canDestroyInput = false)
   {
 #  ifndef ITK_USE_CUFFTW
     const std::lock_guard<FFTWGlobalConfiguration::MutexType> lockGuard(FFTWGlobalConfiguration::GetLockMutex());
     fftwf_plan_with_nthreads(threads);
```

Comment:
Requested changes (3) because the `threads` parameter is actually used here! Please use `[[maybe_unused]]` in this case!